{
  "cve": "CVE-2022-4396",
  "mitre": {
    "cpes": [],
    "created": "2022-12-10T00:00:00+00:00",
    "description": "A vulnerability was found in RDFlib pyrdfa3 and classified as problematic. This issue affects the function _get_option of the file pyRdfa/__init__.py. The manipulation leads to cross site scripting. The attack may be initiated remotely. The name of the patch is ffd1d62dd50d5f4190013b39cedcdfbd81f3ce3e. It is recommended to apply a patch to fix this issue. The identifier VDB-215249 was assigned to this vulnerability. NOTE: This vulnerability only affects products that are no longer supported by the maintainer.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 3.5,
        "vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:N/I:L/A:N"
      },
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2022/4xxx/CVE-2022-4396.json",
    "references": [
      "https://github.com/RDFLib/pyrdfa3/commit/ffd1d62dd50d5f4190013b39cedcdfbd81f3ce3e",
      "https://github.com/RDFLib/pyrdfa3/pull/40",
      "https://vuldb.com/?id.215249"
    ],
    "title": "RDFlib pyrdfa3 __init__.py _get_option cross site scripting",
    "updated": "2024-08-03T01:41:44.749000+00:00",
    "vendors": [],
    "weaknesses": [
      "CWE-707"
    ]
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:a:pyrdfa3_project:pyrdfa3:-:*:*:*:*:python:*:*"
    ],
    "created": "2022-12-10T12:15:10.797000+00:00",
    "description": "A vulnerability was found in RDFlib pyrdfa3 and classified as problematic. This issue affects the function _get_option of the file pyRdfa/__init__.py. The manipulation leads to cross site scripting. The attack may be initiated remotely. The name of the patch is ffd1d62dd50d5f4190013b39cedcdfbd81f3ce3e. It is recommended to apply a patch to fix this issue. The identifier VDB-215249 was assigned to this vulnerability. NOTE: This vulnerability only affects products that are no longer supported by the maintainer.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 5.4,
        "vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:C/C:L/I:L/A:N"
      },
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2022/CVE-2022-4396.json",
    "references": [
      "https://github.com/RDFLib/pyrdfa3/commit/ffd1d62dd50d5f4190013b39cedcdfbd81f3ce3e",
      "https://github.com/RDFLib/pyrdfa3/pull/40",
      "https://vuldb.com/?id.215249"
    ],
    "title": null,
    "updated": "2024-08-03T02:16:17.623000+00:00",
    "vendors": [
      "pyrdfa3_project",
      "pyrdfa3_project$PRODUCT$pyrdfa3"
    ],
    "weaknesses": [
      "CWE-707",
      "CWE-79"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [
        "cpe:2.3:a:pyrdfa3_project:pyrdfa3:-:*:*:*:*:python:*:*"
      ],
      "providers": [
        "nvd"
      ]
    },
    "created": {
      "data": "2022-12-10T00:00:00+00:00",
      "provider": "mitre"
    },
    "description": {
      "data": "A vulnerability was found in RDFlib pyrdfa3 and classified as problematic. This issue affects the function _get_option of the file pyRdfa/__init__.py. The manipulation leads to cross site scripting. The attack may be initiated remotely. The name of the patch is ffd1d62dd50d5f4190013b39cedcdfbd81f3ce3e. It is recommended to apply a patch to fix this issue. The identifier VDB-215249 was assigned to this vulnerability. NOTE: This vulnerability only affects products that are no longer supported by the maintainer.",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_1": {
        "data": {
          "score": 3.5,
          "vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:N/I:L/A:N"
        },
        "provider": "mitre"
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": null,
        "provider": null
      }
    },
    "references": {
      "data": [
        "https://github.com/RDFLib/pyrdfa3/commit/ffd1d62dd50d5f4190013b39cedcdfbd81f3ce3e",
        "https://github.com/RDFLib/pyrdfa3/pull/40",
        "https://vuldb.com/?id.215249"
      ],
      "providers": [
        "mitre",
        "nvd"
      ]
    },
    "title": {
      "data": "RDFlib pyrdfa3 __init__.py _get_option cross site scripting",
      "provider": "mitre"
    },
    "updated": {
      "data": "2024-08-03T02:16:17.623000+00:00",
      "provider": "nvd"
    },
    "vendors": {
      "data": [
        "pyrdfa3_project",
        "pyrdfa3_project$PRODUCT$pyrdfa3"
      ],
      "providers": [
        "nvd"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-707",
        "CWE-79"
      ],
      "providers": [
        "mitre",
        "nvd"
      ]
    }
  }
}